modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/PartitionCommand.java

```diff
@@ -28,4 +28,9 @@ public interface PartitionCommand extends SafeTimePropagatingCommand {
      * Returns a transaction id.
      */
     UUID txId();
+
+    /**
+     * Returns {@code true} if a command represents a full transaction.
+     */
+    boolean full();
```

Comment:
I haven't figured out better name.
What naming do you propose ?

Reply:
onePhaseTxn ?

Reply:
onePhaseTransaction is not very good to me.
full means "a full transaction" - including all keys. 
the txn is considered one-phase if it fits in one replication command.
I prefer to keep full for now. We can rename it later at any moment.

Reply:
I agree with Alexander, first time I saw "full" I thought it means something like "full-process tx", which is quite opposite to actual meaning. Maybe "complete" would be better? Or at least you should explain it in javadoc, because it is really counter-intuitive.

Reply:
I've considerer "complete" and didn't like it either. I think there is already an explanation in javadoc.

Reply:
you provided better explanation in the comment above. Why not just to put something similar to the javadoc? Current one is not elaborated enough.